guillotina/tests/test_errors.py

```diff
@@ -29,3 +32,25 @@ def test_value_serialization_error():
     error = ValueDeserializationError("Foo", "Bar", "Something wrong")
     assert error.field == "Foo"
     assert error.value == "Bar"
+
+
+async def test_handle_cancelled_error(container_requester):
+    async with container_requester as requester:
+        with mock.patch("guillotina.api.content.DefaultGET.__call__") as handle_mock:  # noqa
+            f = asyncio.Future()
+            f.set_result(None)
+            handle_mock.return_value = f
+            handle_mock.side_effect = asyncio.CancelledError()
+            response, status = await requester("GET", "/db/guillotina")
+            status == 499
```

Comment:
You forgot to assert :P I didn't understand why the coverage decreased...

Reply:
eeek. I think we have multiple tests with this. I'll check again.

Reply:
fixed here: https://github.com/plone/guillotina/pull/786